test/models/bot/smooch_test.rb

```diff
@@ -69,9 +69,11 @@ def setup
     @media_url = 'https://smooch.com/image/test.jpeg'
     @media_url_2 = 'https://smooch.com/image/test2.jpeg'
     @video_url = 'https://smooch.com/video/test.mp4'
+    @video_ur_2 = 'https://smooch.com/video/test2.mp4'
```

Comment:
is this a file that's not really a video?

Reply:
Done